subsys/mpsl/Kconfig

```diff
@@ -6,7 +6,6 @@
 
 config MPSL_FEM_ONLY
 	bool "Support only radio front-end module (FEM)"
-	depends on MPSL_FEM
```

Comment:
Why is it possible to remove the dependency to MPSL_FEM here?

Reply:
MPSL has also some kind of fem none implementation which can be used if there is not any fem in the dts to avoid conditional compilation and make protocol logic simpler. It is possible to use FEM API even if MPSL_FEM is disabled

Reply:
The other radio protocol uses the same mechanism like 802.15.4 and so on. @ahasztag But please check if  the FEM API is used in valid way here

Reply:
I don't think removing the dependency is a good idea. `MPSL_FEM` is only enabled if any of the FEM's is configured in device tree. If `MPSL_FEM_ONLY` doesn't depend on `MPSL_FEM` all these checks are lost.

I don't see how conditional compilation would be needed, if `CONFIG_MPSL_FEM_ONLY` is not set. Could you elaborate a bit more on that?

Reply:
Ok, after a quick call with @KAGA164  everything is clear now.

The dependency on `MPSL_FEM` is not needed. The only problem is that the `MPSL_FEM_ONLY` Kconfig option name is misleading. A better name would be simply `MPSL_NO_INIT`. However the name can be changed in the different PR.

Reply:
I agree we can check if we could change `MPSL_FEM_ONLY` to more meaningful name